componentified-todo-list/amirmahdi/to-do-list/src/Itemlist.js

```diff
@@ -1,6 +1,3 @@
-import React from 'react';
```

Comment:
__[HIGH]__  
You need `React` import for using jsx. Just remove the css import.